Common/cpp/Fabric/PropsRegistry.h

```diff
@@ -33,6 +33,10 @@ class PropsRegistry {
     return letMeIn_.exchange(false);
   }
 
+  void resetSkipCommitFlag() {
+    letMeIn_ = false;
```

Comment:
I think it's high time to rename this flag, what do you think about `shouldReanimatedSkipCommit_`?